pkg/action/install.go

```diff
@@ -145,20 +145,24 @@ func (i *Install) installCRDs(crds []chart.CRD) error {
 		}
 		totalItems = append(totalItems, res...)
 	}
-	// Invalidate the local cache, since it will not have the new CRDs
-	// present.
-	discoveryClient, err := i.cfg.RESTClientGetter.ToDiscoveryClient()
-	if err != nil {
-		return err
-	}
-	i.cfg.Log("Clearing discovery cache")
-	discoveryClient.Invalidate()
-	// Give time for the CRD to be recognized.
-	if err := i.cfg.KubeClient.Wait(totalItems, 60*time.Second); err != nil {
-		return err
+	if len(totalItems) > 0 {
```

Comment:
I do not understand this change. Is this for performance? I am worried that if another process deletes a CRD, the local cache could be outdated here.

Reply:
When testing, the `res` returned by `i.cfg.KubeClient.Build(bytes.NewBuffer(obj.File.Data), false)` is coming as `nil`, and the `err` is nil as well which means that there is no issue with building the CRDs and installing them. When `res` returns as `nil` it means that the builder used was not able to get the kube cache here. This is even reflected in this change I have made here. If the length of totalItems is 0 and still the CRD is applied then the Cache is not present on the machine under $HOME/.kube/cache/discovery.

Reply:
Sounds fine. I'll restate my LGTM for this PR.

Reply:
Thanks Matt !